commands.md

```diff
@@ -7,8 +7,12 @@ Command | Implemented By | Who can run it | Description
 `/assign [@userA @userB @etc]` | prow [assign](./prow/plugins/assign) | kubernetes org members | Assigns specified people (or yourself if no one is specified)
 `/unassign [@userA @userB @etc]` | prow [assign](./prow/plugins/assign) | kubernetes org members | Unassigns specified people (or yourself if no one is specified)
 `/area` | prow [label](./prow/plugins/label) | anyone | adds an area/<> label if it exists
+`/remove-area` | prow [label](./prow/plugins/label) | anyone | removes an area/<> label if it exists
```

Comment:
I read the code and I understand.  Make it clear in the docs that these commands take arg(s) that are the label suffixes.